_data/chapters.yaml

```diff
@@ -52,28 +48,26 @@ chapters:
     url: /bay-area
     activity_level: active
   - text: South Florida
-    twitter: southfltechworkers
     activity_level: inactive
   - text: Bangalore
     twitter: TechiesSpeakUp
     url: https://twitter.com/twc_bangalore
     activity_level: inactive
   - text: Berlin
     twitter: TechWorkersBER
-    facebook: TechWorkersBER
+    # facebook: TechWorkersBER # broken link, what's the real one?
```

Comment:
Is there an active version of this link? I could not find it with FB search. cc @chobeat